Dockerfile

```diff
@@ -1,10 +1,30 @@
-FROM rust:1.61 as builder
-RUN rustup target add x86_64-unknown-linux-musl
-RUN apt-get update && apt-get install --no-install-recommends -y musl-tools
+FROM --platform=$BUILDPLATFORM rust:1.61 AS vendor
```

Comment:
This stage also allows some caching on the dependencies. As long as we don't change the `Cargo.{toml,lock}`, this won't have to be rebuilt and will improve the build time.